src/SFA.DAS.EmployerFinance/DependencyResolution/ManageApprenticeshipsOuterApiRegistry.cs

```diff
@@ -12,6 +13,7 @@ public ManageApprenticeshipsOuterApiRegistry ()
         {
             For<ManageApprenticeshipsOuterApiConfiguration>().Use(c => c.GetInstance<EmployerFinanceConfiguration>().ManageApprenticeshipsOuterApiConfiguration).Singleton();
             For<IApiClient>().Use<ApiClient>().Ctor<HttpClient>().Is(new HttpClient()).Singleton();
+            For<ICohortsService>().Use<CohortsService>().Ctor<HttpClient>().Is(new HttpClient()).Ctor<string>().Is(c => c.GetInstance<EmployerFinanceConfiguration>().EmployerCommitmentsBaseUrl).Singleton();
```

Comment:
If we can use the same pattern as LevyTransferMatchingService maybe it can be injected the same way?